dashboard/app/controllers/api/v1/ml_models_controller.rb

```diff
@@ -13,7 +13,13 @@ def save
     model_id = generate_id
     model_data = params["ml_model"]
     return head :bad_request if model_data.nil? || model_data == ""
-    profanity_or_pii = ShareFiltering.find_failure(model_data.to_s, request.locale)
+    begin
+      profanity_or_pii = ShareFiltering.find_failure(
+        model_data.except(:trainedModel).to_s, request.locale
+      )
+    rescue OpenURI::HTTPError => e
+      print e
```

Comment:
What will the user see in this case?

Reply:
It was just a way to show something in the server log. With the most recent push, we're now logging any PII/profanity API errors with Firehose. If there is an error, the user can still save a model but it won't be filtered.